src/sinks/prometheus.rs

```diff
@@ -172,8 +201,17 @@ impl Sink for PrometheusSink {
                     Some(Direction::Minus) => gauge.sub(val),
                 }
             }),
-            _ => {
-                // TODO: support all the metric types
+            Metric::Histogram {
+                name,
+                val,
+                sample_rate,
+            } => self.with_histogram(name, |hist| {
+                for _ in 0..sample_rate {
+                    hist.observe(val as f64);
+                }
+            }),
+            Metric::Set { .. } => {
+                info!("Sets are not supported in Prometheus sink");
```

Comment:
I would either make this a `trace!` level thing or a rate-limited `warn!`. I don't want to log-spam users on every `Metric::Set` that comes through.

I was also wondering if we could support sets by exposing them as a gauge/counter value. We don't need to do it now, but could be interesting for the future.